plugin/server/services/format.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+const showdown = require('showdown')
+const jsdom = require('jsdom')
+
+const dom = new jsdom.JSDOM()
+const showdownConverter = new showdown.Converter({
+  noHeaderId: true,
+  strikethrough: true,
+})
+
+function markdownToHtml(singleText) {
+  return showdownConverter.makeHtml(singleText)
+}
+
+function htmlToMarkdown(singleText) {
+  return showdownConverter
+    .makeMarkdown(singleText, dom.window.document)
+    .replace('<!-- -->\n', '')
```

Comment:
I think this should be `replaceAll` instead?

Reply:
Not supported in node 14, using `.replace(/<!-- -->\n/g, '')` instead